entrypoint.sh

```diff
@@ -3,14 +3,17 @@
 TEST_SUITE_NAME=$1
 ENABLE_DEBUG=$2
 CLUSTER_NAME=$3
-
+TESTKUBE_VERSION=$4
 set -e
 
 # Enable debugging output
 if [[ "${ENABLE_DEBUG}" == "true" ]]; then
     set -x
 fi
 
+# Install testkube plugin
+echo "Installing testkube specific version"
+curl -sSLf https://kubeshop.github.io/testkube/install.sh | bash
```

Comment:
The testkube is installed on the entry point side with the version defined in the env var: $TESTKUBE_VERSION

Reply:
While it is nice to be able to define the version on the fly for each run it is not so nice that you have to download this binary and run the install every single time the action runs.  

Please put this download logic into the Dockerfile.

While you can't take user input params for the Dockerfile, peg the version there and we will do releases for the versions we want.

Reply:
Hi @sekka1 , one question: are you ok with the version number being hardcoded in the dockerfile, as I don't think I have a way to get an input variable to that point. or I have?

Reply:
Hi @sekka1 , your request was attended 👍 
new workflow run of evidence proof: https://github.com/exact-payments/gruntwork-infrastructure-live/runs/5758570897?check_suite_focus=true#step:13:25